packages/orchestrator/internal/sandbox/sandbox.go

```diff
@@ -443,3 +444,41 @@ func (s *Sandbox) SlotIdx() int {
 func (s *Sandbox) FcPid() int {
 	return s.fc.pid
 }
+
+type SandboxStats struct {
+	CPUPercent float64 `json:"cpu_percent"`
+	MemoryVMS  uint64  `json:"memory_vms"`
+	MemoryRSS  uint64  `json:"memory_rss"`
+}
+
+func (s *Sandbox) Stats() (SandboxStats, error) {
+	stats := SandboxStats{
+		CPUPercent: 0,
```

Comment:
Are you explicitly initializing the struct field for clarity?

Reply:
It's that Go extension that kept bugging me about initializing the values, I don't usually do it.